tests/core/test_safeguards.py

```
@@ -0,0 +1,414 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)